elado/pannonhalma/index.md

```diff
@@ -0,0 +1,121 @@
+
+layout: elado
+title:  Vadregényes vízi-világ eladó Pannonhalmán 
+
+
+**Irányára: 3500000 €**
+---
+
+#![Kívülről](http://i.imgur.com/J4m9gPe.jpg) Vadregényes vízi-világ eladó Pannonhalmán
+
+{% include fold.html %}
+
+Eladó Pannonhalmán egy 80 ha-os, vadregényes ingatlan. 
+
+A birtokot a Világörökség részét képező Pannonhalmi Bencés Apátság impozáns épület-együttese őrzi. A tulajdonos család szorgalmát, igényességét dicséri a hatalmas, mindenhol gondozott terület.
+
+![Kívülről](http://i.imgur.com/gxLCTPr.jpg)
```

Comment:
Minden képre azt állítottad be címnek, hogy `Kívülről`. Ez végülis nem baj, csak akkor van szerepe azt hiszem, ha valami gyengénlátó és felolvastatja az oldalt egy programmmal. Akkor a program ezt a címet fogja a képhez felolvasni.

Szerintem jobb semmilyen címet sem megadni, vagy írj be valami valósat. Az utóbbi neked is segíthet, mert amikor szerkeszted az oldalt, rögtön látod, melyik kép melyik a leírásából. Például ha változtatni akarod a képek sorrendjét, ez hasznos lehet.